docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -213,6 +229,53 @@ following format:
   !<InstructionMetadata1> = !{<Operand1>, <Operand2>, ..}
   !<InstructionMetadata2> = !{<Operand1>, <Operand2>, ..}
 
++--------------------+---------------------------------------------------------+
+| SPIR-V instruction | LLVM IR                                                 |
++====================+=========================================================+
+| OpSource           | .. code-block:: llvm                                    |
+|                    |                                                         |
+|                    |    !spirv.Source = !{!0}                                |
+|                    |    !0 = !{i32 3, i32 102000, !1}                        |
+|                    |    ; 3 - OpenCL_C                                       |
+|                    |    ; 102000 - OpenCL version 1.2                        |
+|                    |    ; !1 - optional file id.                             |
+|                    |    !1 = !{!"/tmp/opencl/program.cl"}                    |
++--------------------+---------------------------------------------------------+
+| OpSourceExtension  | .. code-block:: llvm                                    |
+|                    |                                                         |
+|                    |    !spirv.SourceExtension = !{!0, !1}                   |
+|                    |    !0 = !{!"cl_khr_fp16"}                               |
+|                    |    !1 = !{!"cl_khr_gl_sharing"}                         |
++--------------------+---------------------------------------------------------+
+| OpExtension        | .. code-block:: llvm                                    |
+|                    |                                                         |
+|                    |    !spirv.Extension = !{!0}                             |
+|                    |    !0 = !{!"cl_khr_fp16"}                               |
```

Comment:
`cl_khr_fp16` is a _source_ extension. I suggest using a SPIR-V extension, for example:
```suggestion
|                    |    !0 = !{!"SPV_KHR_expect_assume"}                               |
```